torchvision/prototype/transforms/_misc.py

```diff
@@ -32,15 +34,17 @@ def extra_repr(self) -> str:
 
 
 class Normalize(Transform):
-    _DISPATCHER = F.normalize
-
     def __init__(self, mean: List[float], std: List[float]):
         super().__init__()
         self.mean = mean
         self.std = std
 
-    def _get_params(self, sample: Any) -> Dict[str, Any]:
-        return dict(mean=self.mean, std=self.std)
+    def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
+        if type(input) is torch.Tensor:
+            return _F.normalize(input, mean=self.mean, std=self.std)
+        if type(input) is features.Image:
+            output = K.normalize_image(input, mean=self.mean, std=self.std)
+            return features.Image.new_like(input, output)
```

Comment:
Hm... Should it be an image after Normalization? The colour scheme will not be valid anymore (RGB 0-255 assumption). I think we should be returning a Tensor here, not a `features.Image`.

Reply:
That is one possibility. Another would be to use 

https://github.com/pytorch/vision/blob/e88a55493eadaab15d1df432fe248e70e04d29e4/torchvision/prototype/features/_image.py#L16-L17

Anyway, I agree, the output should not be an RGB image.

Reply:
There are some PyTorch features (such as the new FLOP estimation) that we will not be able to use if we subclass tensor. Let's remove the `Image` wrapping for now.